pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```diff
@@ -45,15 +45,18 @@ func TestNodeAffinity(t *testing.T) {
 		disablePreFilter    bool
 	}{
 		{
-			name: "no selector",
-			pod:  &v1.Pod{},
+			name:                "no selector and affinity",
+			pod:                 &v1.Pod{},
+			wantPreFilterStatus: framework.NewStatus(framework.Skip),
+			wantStatus:          nil, // Actually, the scheduler will skip NodeAffinity Filter as PreFilter returns Skip.
 		},
 		{
 			name: "missing labels",
 			pod: st.MakePod().NodeSelector(map[string]string{
 				"foo": "bar",
 			}).Obj(),
-			wantStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonPod),
+			wantStatus:       framework.NewStatus(framework.UnschedulableAndUnresolvable, ErrReasonPod),
+			disablePreFilter: true,
```

Comment:
why? Same question for the rest of the cases.

Reply:
Removed all `disablePreFilter`.